middleware/error.js

```diff
@@ -0,0 +1,29 @@
+const ErrorResponse = require('../utils/errorResponse');
+
+const errorHandler = (err, req, res, next) => {
+  let error = { ...err };
+
+  error.message = err.message;
```

Comment:
This is very likely redundant. Since `error = {...err}` would have done that already (copying the object).

Reply:
done